terraform/deployments/trusted-setup/participants.tf

```diff
@@ -0,0 +1,11 @@
+/*
+module "<participant>" {
+  source = "../../../modules/trusted-setup-participant/"
+
+  server_id = "${aws_transfer_server.challenge_transfer.id}"
+  role      = "${aws_iam_role.participant.arn}"
+  bucket    = "${aws_s3_bucket.trusted_setup.bucket}"
+  user_name = "<participant>"
+  public_key = "${file("keys/<participant>.pub")}"
+}
+*/
```

Comment:
Not sure if this is actually what we want to do. We'd have to create a module for every participant, and couldn't use any lists as module doesn't support count.

It might be better to just put this directly into `deployments/trusted-setup` so we can define participants as a list?